azure-ts-aks-multicluster/Pulumi.yaml

```diff
@@ -0,0 +1,14 @@
+name: azure-ts-aks-multicluster
+runtime: nodejs
+description: Create multiple Azure Kubernetes Service (AKS) clusters in different regions and with different node counts.
+template:
+  description: Create multiple Azure Kubernetes Service (AKS) clusters in different regions and with different node counts.
```

Comment:
This line can now be deleted (we don't need to duplicate the `description` under `template` anymore).

Reply:
Removed. Thanks!